index.html

```diff
@@ -5,14 +5,16 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./styles.css" />
-    <title>gehraiyaan</title>
+    <title>Home | gehraiyaan</title>
   </head>
   <body>
     <div class="quiz-container">
       <nav class="quiz-navbar flex flex-align-center justify-between">
-        <div class="nav-logo flex justify-center">gehraaiyaan</div>
+        <div class="nav-logo flex justify-center">
+          <a href="./index.html"> gehraaiyaan </a>
+        </div>
         <div class="nav-right">
-          <button class="btn">
+          <button class="btn primary-btn">
             <p>Signup</p>
             <div>
               <img class="svg" src="/assets/svg/angle-right-solid.svg" />
```

Comment:
add alt attribute to img

Reply:
That is an SVG I've added locally and so there is no alt text.